### ISSUE TITLE
About Adjustable Gradient Clipping

### ISSUE THREAD

**Issue by kweisamx** [Jongchan/tensorflow-vdsr#27]
https://github.com/Jongchan/tensorflow-vdsr/blob/master/MODEL.py#L10

May I ask why the line is mean that has the adjustable gradient clipping?
Or where you do this in other code?

Thanks.

**Comment by Jongchan** [comment]
I think there are some misunderstandings. That line does not mean gradient clipping. Maybe it is mixed up with other commits. I did implement gradient clipping in the first commit of this repository, but now it is removed.
If you look at the old commits, you can find the corresponding codes.

I removed gradient clipping because Adam optimizer can automatically keep the loss from diverging.
